src/modules/callouts/pages/CalloutsAdminPage.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <div class="mb-5 flex justify-between border-primary-40 border-b pb-3">
+    <PageTitle :title="t('menu.callouts')"></PageTitle>
+    <div class="flex-1 md:hidden">
+      <AppSelect v-model="currentStatus" :items="statusItems" />
+    </div>
+    <div class="flex-0 ml-3">
+      <AppButton href="/tools/polls">{{
+        t('calloutsAdmin.addCallout')
+      }}</AppButton>
+    </div>
+  </div>
+  <div class="md:flex">
```

Comment:
The `flex` here is a little bit out whack from ~1,080px down. 

Now, I remember "desktop-first for admins", but just checking with @beabee-communityrm/design that this needs fixing indeed.

Reply:
The filters on the left become a dropdown below 768px. They could switch to dropdown at <1080px to make more space for the table. Do you think that would work?